picard_plugin_tools/__init__.py

```diff
@@ -290,16 +295,16 @@ def _create_manifest(manifest_path, manifest_data=None, missing_fields=None):
     return manifest_data
 
 
-@cli.command()
+@cli.command('create_manifest')
```

Comment:
GNU standards for command line options usually uses '-' instead of '_' for long options, so i think it would be a good idea to replace this with: `@cli.command('create-manifest')`
Same goes for other commands of course.

https://www.gnu.org/prep/standards/html_node/Option-Table.html#Option-Table